src/containers/feed/new-post/new-post.jsx

```diff
@@ -11,11 +11,13 @@ import * as actions from './new-post.actions';
 
 import styles from './new-post.less';
 
-const NewPostContainer = ({ state, postEditor, openDialog, closeDialog, closeErrorDialog, submitPostForm, suggest }) => (
+const NewPostContainer = ({ state, postEditor, openDialog, closeDialog, closeErrorDialog, submitPostForm, suggest, isAuthorized }) => (
     <div>
-        <FloatingActionButton title={title(suggest)} onTouchTap={openDialog} className={styles['new-post-button']}>
-            <Create />
-        </FloatingActionButton>
+        {isAuthorized && (
```

Comment:
check already exists in [feed.jsx](https://github.com/kitos/web-purple/blob/master/src/containers/feed/feed.jsx)